k4e-agent-sti.sh

```diff
@@ -37,10 +88,13 @@ git clone https://github.com/jakub-dzon/yggdrasil.git /home/builder/yggdrasil
 cd /home/builder/yggdrasil
 export CGO_ENABLED=0
 
+# TODO: Remove, workaround to assure nftables is running
```

Comment:
Need to open a bug

Reply:
worked around it by disabling the firewalld.